packages/blitz-rpc/package.json

```diff
@@ -35,18 +36,20 @@
     "supports-color": "8.1.1"
   },
   "peerDependencies": {
+    "@tanstack/query-core": "4.24.4",
```

Comment:
@flybayer I don't think this is necessary since we have `react-query` as the dependency which in turn has `@tanstack/query-core` as its dependency. So I feel this being in peer-deps would be better mainly for pnpm. 
Do you think its better to move to deps anyway?